exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,20 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80){
+    return "A";
+  }
+  else if(mark <80 && mark > 60){
+    return "B";
+  }
+  else if(mark <= 60 && mark >= 50){
+    return "C";
+  }
+  else{
+    return "F";
+  }
```

Comment:
What you wrote is correct, but because we don't run any code after return, you could rewrite it without any else if or else. Can you think about why this code is still working correctly?
```
if (mark >= 80) {
    return "A";
 }
 if (mark < 80 && mark > 60) {
    return "B";
  } 
  if (mark <= 60 && mark >= 50) {
    return "C";
  }
  return "F"; 
```